insights/specs/default.py

```diff
@@ -300,7 +300,7 @@ def gfs2_mount_points(broker):
     grubby_default_index = simple_command("/usr/sbin/grubby --default-index")  # only RHEL7 and updwards
     grubby_default_kernel = simple_command("/sbin/grubby --default-kernel")
     hammer_task_list = simple_command("/usr/bin/hammer --config /root/.hammer/cli.modules.d/foreman.yml --output csv task list --search 'state=running AND ( label=Actions::Candlepin::ListenOnCandlepinEvents OR label=Actions::Katello::EventQueue::Monitor )'")
-    haproxy_cfg = first_file(["/var/lib/config-data/puppet-generated/haproxy/etc/haproxy/haproxy.cfg", "/etc/haproxy/haproxy.cfg"])
+    haproxy_cfg = first_file(["/var/lib/config-data/puppet-generated/haproxy/etc/haproxy/haproxy.cfg", "/etc/haproxy/haproxy.cfg", "/etc/opt/rh/rh-haproxy18/haproxy/haproxy.cfg"])
```

Comment:
@jobselko would the scl haproxy config path need to be in front of the normal config path in case they have both installed?

Reply:
No, the order is not important. I just did not know the list is processed in reverse order.

Reply:
@jobselko Okay cool thank you for confirming that.

Reply:
@ryan-blakley And what about the order? Is it OK?

Reply:
@jobselko The order is fine if the order doesn't matter. My concern was if they have the regular haproxy package installed along with the software collections haproxy package installed. Then it would only parse the regular haproxy package's config file since it's in the list before the software collections haproxy package's config file.

Reply:
@ryan-blakley Now I see a problem. The situation you are describing could occur. But even if we switch the order of that files in the list, the problem is still here. Another solution would be to collect both files (have two different _first_file_ or _use glob_file_).
I will try to find more info tomorrow.

Reply:
@jobselko I thought that you should never encounter both, as it is either RHEL-8 where it is available as a normal package, or RHEL-7 where it is SCL. Is there a way to get combination somehow?

Reply:
All config files for both regular and SCL packages should be collected. I adjusted the code.
@ryan-blakley Please review.

Reply:
@skontar on rhel 7 there is a haproxy rpm in the main repo, and there is rh-haproxy18 rpm in the rhscl repo, I just tested to make sure, both can be installed at the same time. 

@jobselko if using glob the parser would need to be updated due to it using multi-collect.